aospy/calc.py

```diff
@@ -294,10 +332,9 @@ def _add_grid_attributes(self, ds):
                 self.pressure = ds.level
         return ds
 
+    # @log_step(logging.info, "Getting input data: {}".format(self.var))
```

Comment:
Were calls like this generating name errors for `self.var`?

Reply:
Basically yes.  I got a little too cute with trying to switch the logging to be all in terms of decorators (and which is out of scope for this PR anyways).  For those logging messages that rely on the method's arguments, we'll just have to retain the within-method logging call.

One could stay with the pure decorator approach by creating wrapper methods, but that seems to be more trouble than its worth.  I might just revert all of these logging changes, as they probably deserve their own consideration and PR.